js/deer-record.js

```diff
@@ -226,7 +227,7 @@ export default class DeerReport {
     processRecord(event) {
         event.preventDefault()
         this.evidence = this.elem.getAttribute(DEER.EVIDENCE) // inherited to inputs
```

Comment:
Honestly, I think all these are redundant, but that's not what we're looking at today.

Reply:
Yeah, that can be cleanup.